pkg/operator/ceph/controller/spec.go

```diff
@@ -106,11 +106,16 @@ PERIODICITY=%s
 LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
 LOG_MAX_SIZE=%s
 ROTATE=%s
+ADDITIONAL_LOG_FILE=%s
```

Comment:
The calling function uses a space-separated list of files, so let's make it clear here that it could be multiple

```suggestion
ADDITIONAL_LOG_FILES=%s
```